pages/claim/[type]/[code].tsx

```diff
@@ -94,20 +94,21 @@ function ClaimDonation({}: Props): ReactElement {
       code: code,
     };
     if (contextLoaded && user) {
-      const userLang = localStorage.getItem('language') || 'en';
       postAuthenticatedRequest(
-        `/api/v1.3/${userLang}/validateCode`,
+        `/app/redeem`,
         submitData,
         token,
         handleError
       ).then((res) => {
         if (res.code === 401) {
           setErrorMessage(res.message);
           setIsUploadingData(false);
-        } else if (res.status === 'error') {
-          setErrorMessage(res.errorText || t('me:wentWrong'));
+        } else if (res.error_code === 'already_redeemed') {
+          setErrorMessage(res.error_code || t('me:wentWrong'));
           setIsUploadingData(false);
-        } else if (res.status === 'success') {
+        } else if (res.error_code === 'invalid_code') {
+          setErrorMessage(res.error_code);
```

Comment:
errorMessage text needs to be translated. Currently the key (e.g. 'already_redeemed') will display instead of the desired text.